qemu/tests/migration.py

```diff
@@ -165,6 +165,9 @@ def guest_stress_deamon():
     check = params.get("vmstate_check", "no") == "yes"
     living_guest_os = params.get("migration_living_guest", "yes") == "yes"
     deamon_thread = None
+    depends_pkgs = params.objects("depends_pkgs")
+    test.log.info("Install packages: %s in host", depends_pkgs)
+    utils_package.package_install(depends_pkgs)
```

Comment:
Need check the status, cancel test when install pkg failed.   Otherwise, test will continue and fail as missing command finally.